src/aiq/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```diff
@@ -136,6 +137,21 @@ def set_cors_config(self, aiq_app: FastAPI) -> None:
             **cors_kwargs,
         )
 
+    def set_user_attributes(self, request: Request, session_manager: AIQSessionManager) -> None:
+        """
+        Extracts and sets user request attributes from an HTTP request.
+        """
+        session_manager.context.user_attributes._request.method = request.method
```

Comment:
We do not want to set private methods like this. This should get set in the `AIQSessionManager.session()` function. Add a new argument `request` to this method.